fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java

```diff
@@ -0,0 +1,45 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.google.common.io.FileBackedOutputStream;
+import com.google.gson.Gson;
+import com.starrocks.analysis.AccessTestUtil;
+import com.starrocks.analysis.Analyzer;
+import com.starrocks.persist.gson.GsonUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JDBCResourceTest {
+    private Analyzer analyzer;
+
+    @Before
+    public void setUp() {
+        analyzer = AccessTestUtil.fetchAdminAnalyzer(true);
+    }
+
+    @Test
+    public void testSerialization() throws Exception {
```

Comment:
should test the exception case. for example， the config key is 'xxxx'

Reply:
@kangkaisen fixed